script.js

```diff
@@ -0,0 +1,11 @@
+const modal = document.querySelector(".modal");
+const trigger = document.querySelector(".trigger");
```

Comment:
does this need its own variable? your only using it once, variable seems like a waste of time